scripts/run_integration_tests.sh

```diff
@@ -32,8 +32,9 @@ mismatch_found=false
 node16=("nodejs16.x" "16.14" $(xxd -l 4 -c 4 -p < /dev/random))
 node18=("nodejs18.x" "18.12" $(xxd -l 4 -c 4 -p < /dev/random))
 node20=("nodejs20.x" "20.9" $(xxd -l 4 -c 4 -p < /dev/random))
```

Comment:
should we be upgrading the other versions as well? the latest ones are 16.20.2, 18.20.5, and 20.18.1 ( https://nodejs.org/en/about/previous-releases#looking-for-latest-release-of-a-version-branch )

Reply:
I guess it depends on what the lambda runtime is using?

Reply:
i didn't see it in the aws docs when i took a peek, so i would guess they upgrade them behind the scenes? i'll do an inventory once this is merged and we have some breathing room =]